### ISSUE TITLE
Adding inside horizontal stack seems to break the card layout

### ISSUE THREAD

**Issue by realroywalker** [jcooper-korg/AlarmPanel#1]
Firstly, thanks for the great card - love the extra features this provides!
I'm currently trying to add it to my tablet layout so that I can display the alarm card side by side with multiple-entity-row card to display door/window statuses.

However currently when I add the alarm card everything on the page seems to want to fit to one column, and if I place alarm card inside horizontal stack (along with multiple-entity-row) the alarm card layout breaks and everything seems squashed to the left.

Is what i'm trying to achieve possible, using this custom card?

**Comment by jcooper-korg** [comment]
Hey- thanks for the feedback!  
Please attach your dashboard.yaml and some screenshots and I'll take a look

**Comment by realroywalker** [comment]
Hi - I just went on to grab the yaml and realised I've been an idiot.. I didn't have panel mode switched on for the view, so everything was in one narrow column.
I thought it was the alarm card that was the issue, as the layout of the card was totally broken (buttons overlapping the background edges etc.) then I twigged it was because the available space simply wasn't wide enough to display everything, so it was overflowing.

With panel mode on, it works a treat! - Thanks for the great card 👍
